### PR TITLE
Add plugin integrations to docs

### PR DESCRIPTION
**Pull request by karenzone**
Dependencies:
- [x] Integration asciidoc files must be present
   - [x] #803
   - [x] #806
   - [x] #807

### CODE REVIEW

**Comment by karenzone** [comment]
DO NOT MERGE:  Unsatisfied dependencies

**Comment by jsvd** [comment]
@karenzone anything missing here?

**Comment by karenzone** [comment]
@jsvd. Thanks for asking! Yes, still some pieces missing. I'm working to flesh them out.

**Comment by karenzone** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by karenzone** [comment]
@jsvd @yaauie  I've met all the dependencies, so this is ready for review.  
If needed, we can continue to iterate on messaging after I get everything hooked up.

File: docs/plugins/integrations.asciidoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[[plugin-integrations]]
+== Plugin integrations
+
+Plugin integrations combine related plugins--inputs, outputs, and sometimes
+filters and codecs--into one package. 
+
+|=======================================================================
+| Plugin Integration | Description | Github repository
+| <<plugins-integration-kafka,kafka>> | Plugins for use with the Kafka distributed streaming platform. | https://github.com/logstash-plugins/logstash-integration-kafka[logstash-integration-kafka]
+| <<plugins-integration-rabbitmq,rabbitmq>> | Plugins for processing events to or from a RabbitMQ exchange. | https://github.com/logstash-plugins/logstash-integration-rabbitmq[logstash-integration-rabbitmq]
```
Comment (jsvd) [inline docs/plugins/integrations.asciidoc]: If I recall correctly, in RabbitMQ, one reads from queues and sends to exchanges. More on the AMQP 0.9.1 model here: https://www.rabbitmq.com/tutorials/amqp-concepts.html#amqp-model

So maybe "Plugins for processing events to or from a RabbitMQ broker"?

File: docs/plugins/integrations.asciidoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+|=======================================================================
+
+:edit_url: https://github.com/logstash-plugins/logstash-codec-avro/edit/master/docs/index.asciidoc
```
Comment (jsvd) [inline docs/plugins/integrations.asciidoc]:
```suggestion
:edit_url: https://github.com/logstash-plugins/logstash-integration-kafka/edit/master/docs/index.asciidoc
```

File: docs/plugins/integrations.asciidoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+include::integrations/kafka.asciidoc[]
+
+:edit_url: https://github.com/logstash-plugins/logstash-codec-cef/edit/master/docs/index.asciidoc
```
Comment (jsvd) [inline docs/plugins/integrations.asciidoc]:
```suggestion
:edit_url: https://github.com/logstash-plugins/logstash-integration-rabbitmq/edit/master/docs/index.asciidoc
```

**Pull request closed without merging**